common/supplemental/units.xml

```diff
@@ -0,0 +1,247 @@
+		
+		<!-- frequency -->
+		<convertUnit source='hertz' baseUnit='revolution-per-second' factor='1'/>
+		
+		<!-- force -->
+		<convertUnit source='newton' baseUnit='kilogram-meter-per-square-second' factor='1'/>
+		<convertUnit source='pound-force' baseUnit='kilogram-meter-per-square-second' factor='lb2kg * gravity' systems="US UK"/>
+		
+		<!-- pressure -->
+		<convertUnit source='pascal' baseUnit='kilogram-per-meter-square-second' factor='1'/>
+		<convertUnit source='millimeter-of-mercury' baseUnit='kilogram-per-meter-square-second' factor='10132500/760000'/>
+		<convertUnit source='inch-hg' baseUnit='kilogram-per-meter-square-second' factor='3386.389' systems="US UK"/>
+		<convertUnit source='bar' baseUnit='kilogram-per-meter-square-second' factor='100000'/>
+		<convertUnit source='atmosphere' baseUnit='kilogram-per-meter-square-second' factor='101325'/>
+		
+		<!-- energy -->
+		<convertUnit source='electronvolt' baseUnit='kilogram-square-meter-per-square-second' factor='1.602177E-19'/>
+		<convertUnit source='dalton' baseUnit='kilogram-square-meter-per-square-second' factor='1.49241808560E-10'/>
+		<convertUnit source='joule' baseUnit='kilogram-square-meter-per-square-second' factor='1'/>
+		<convertUnit source='calorie' baseUnit='kilogram-square-meter-per-square-second' factor='4.184'/>
+		<convertUnit source='british-thermal-unit' baseUnit='kilogram-square-meter-per-square-second' factor='1055.06' systems="US UK"/>
+		<convertUnit source='foodcalorie' baseUnit='kilogram-square-meter-per-square-second' factor='4184' systems="US UK"/>
+		<convertUnit source='therm-us' baseUnit='kilogram-square-meter-per-square-second' factor='105506000' systems="US"/>
+		
+		<!-- power -->
+		<convertUnit source='watt' baseUnit='kilogram-square-meter-per-cubic-second' factor='1'/>
+		<convertUnit source='horsepower' baseUnit='kilogram-square-meter-per-cubic-second' factor='ft2m * lb2kg * gravity * 550' systems="US UK"/>
+		<convertUnit source='solar-luminosity' baseUnit='kilogram-square-meter-per-cubic-second' factor='3.828E+26'/>
+		
+		<!-- voltage -->
+		<convertUnit source='volt' baseUnit='kilogram-square-meter-per-cubic-second-ampere' factor='1'/>
+		
+		<!-- electric-resistance -->
+		<convertUnit source='ohm' baseUnit='kilogram-square-meter-per-cubic-second-square-ampere' factor='1'/>
+		
+		<!-- area -->
+		<convertUnit source='dunam' baseUnit='square-meter' factor='1000'/>
+		<convertUnit source='acre' baseUnit='square-meter' factor='ft2m*ft2m * 43560' systems="US UK"/>
+		<convertUnit source='hectare' baseUnit='square-meter' factor='10000'/>
+		
+		<!-- volume -->
+		<convertUnit source='teaspoon' baseUnit='cubic-meter' factor='cup2m3/48' systems="US UK"/>
+		<convertUnit source='tablespoon' baseUnit='cubic-meter' factor='cup2m3/16' systems="US UK"/>
+		<convertUnit source='fluid-ounce-imperial' baseUnit='cubic-meter' factor='0.00002841306'/>
+		<convertUnit source='fluid-ounce' baseUnit='cubic-meter' factor='cup2m3/8' systems="US UK"/>
```

Comment:
And shouldn't this have ONLY system="US"?

Reply:
Yes, the UK ounce is slightly different. Will introduce that and use a constant for the imperial volume measures.

Reply:
fixed